nucypher/blockchain/economics.py

```diff
@@ -85,9 +85,12 @@ def __init__(self,
                  # StakingEscrow
                  initial_supply: int,
                  total_supply: int,
-                 staking_coefficient: int,
-                 locked_periods_coefficient: int,
+                 second_phase_coefficient: int,
```

Comment:
```suggestion
                 second_phase_minting_coefficient: int,
```
if for consistency with Issuer.sol (only if my proposed change to this variable below is rejected)

Reply:
used new name
✔️